node-definitions/vyos/VyOS-Router.yaml

```diff
@@ -0,0 +1,54 @@
+id: VyOS-Router
+general:
+  nature: router
+  description: Opensource Router and Firewall Platform
```

Comment:
Might be good to use `|-` and include a bit more details such as a link to the VyOS docs.